data/verb-forms.json

```diff
@@ -254,6 +254,265 @@
                     "ru": "делают"
                 }
             }
-        ]
+        ],
+        "type": "sahih"
```

Comment:
Let's call it `sound` and put on the top of json, after `base`.

Reply:
Why "sound", it's not about sound, but about the class of verbs or type. And this JSON is just a temp file, which we are using due to the absence of a database, it would be generable later, so what's the point of playing with its structure now, we would rearrange it anyway later.

Reply:
I meant that the words should be in English. Instead of `sahih` we should call it `sound` or `healthy`.
I'm not entirely sure about the database, though.
Let's discuss it separately and how do you want to use it.